### PR TITLE
feat: store submission

### PR DESCRIPTION
**Pull request by mateorodriguezcastillo**
Backend implementation of Store Submission.
Types of Changes
- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
Definition of Done Checklist:
- [x] My code follows the code style and structure of this project.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: app/Models/Submission.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        'info',
+        'symptoms',
+        'status',
+    ];
+
+    protected $attributes = [
+        'status' => 'pending',
+    ];
+
+    public function patient()
```
Comment (sojeda) [inline app/Models/Submission.php:30]: tipo

File: app/Transformers/SubmissionTransformer.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+     *
+     * @param  \App\Submission $submission
+     * @return array
+     */
+    public function transform(Submission $submission)
+    {
+        return [
+            'id' => (int) $submission->id,
+            'patient_id' => (int) $submission->patient_id,
+            'doctor_id' => (int) $submission->doctor_id,
```
Comment (sojeda) [inline app/Transformers/SubmissionTransformer.php:35]: Y que vas hacer en el front con esto?

File: app/Models/Submission.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        'patient_id',
+        'doctor_id',
+        'title',
+        'info',
+        'symptoms',
+        'status',
+    ];
+
+    protected $attributes = [
+        'status' => 'pending',
```
Comment (sojeda) [inline app/Models/Submission.php:27]: Deberiamos crear un enum de Status

**Comment by mateorodriguezcastillo** [comment]
Ahh se me agregaron los cambios del rebaseeeee

**Comment by mateorodriguezcastillo** [comment]
image

**Review by sojeda — APPROVED** [review]

**Pull request merged**